src/components/layout/Navbar/index.tsx

```diff
@@ -30,17 +51,27 @@ const Navbar = ({ user }: NavbarProps) => {
   return (
     <header className={styles.header}>
       <div className={styles.content}>
-        <Link href={config.homeRoute} className={styles.icon}>
+        {/* Mobile Navbar Toggle */}
+        <button
+          type="button"
+          className={`${styles.toggleIcon} ${!mobile && styles.hidden}`}
+          onClick={toggleMenu}
+        >
+          <div className={`${styles.bar1} ${menuOpen && styles.open}`} />
```

Comment:
This is something I learned recently and it doesn't change anything functionally tbh but I would prefer using data attributes like we did in the FE interview takehome to indicate the current state of an element because a data-attribute represents a changing property of this tag, rather than a static CSS style and it allows for easy selection and identification of the current state without extra class names.

This would look something like

`<div className={styles.bar1} data-open={menuOpen}>`

```
.bar1[data-open="true"] {
}
```